src/features/ajax/aggregate/index.js

```diff
@@ -75,13 +75,6 @@ export class Aggregate extends AggregateBase {
 
       handle('bstXhrAgg', ['xhr', hash, params, metrics], undefined, FEATURE_NAMES.sessionTrace, ee)
```

Comment:
Bit of a scope creep, but should the deny-list also block this from getting into session trace too?

Reply:
No. We talked about this and we decided that the info being in the session trace was acceptable.